src/App.js

```diff
@@ -60,7 +63,18 @@ class App extends Component {
     const apiKey = this.getApiKey()
 
     if (!apiKey && this.isTableau) {
-      this.redirectToAuth()
+      if (code) {
+        getToken(code)
+        .then(response => {
+          const token = response.data.access_token
+          if (token) {
+            window.localStorage.removeItem('DW-CODE-VERIFIER')
```

Comment:
Let's move the remove and set functionality below into the `util` class and move `DW-CODE-VERIFIER` into a `const`